src/Message.php

```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types = 1);
+
+namespace LanguageServer\Protocol;
+
+class Message
```

Comment:
this seems more part of the implementation than the protocol, should probably replace it with:

```
abstract class Message
{
    public $jsonrpc = '2.0';
}
```

or similar (and implement `RequestMessage`, `ResponseMessage` etc also...

Reply:
It is part of the protocol, see https://github.com/Microsoft/language-server-protocol/blob/gh-pages/specification.md#base-protocol

Reply:
So I think it belongs into this package